integration-tests/tests/update_test.go

```diff
@@ -73,7 +73,8 @@ func (s *updateSuite) TestUpdateToSameReleaseAndChannel(c *check.C) {
 		common.Reboot(c)
 	} else if common.AfterReboot(c) {
 		common.RemoveRebootMark(c)
-		c.Assert(common.GetCurrentUbuntuCoreVersion(c) > common.GetSavedVersion(c),
-			check.Equals, true)
+		currentVersion := common.GetCurrentUbuntuCoreVersion(c)
+		c.Assert(currentVersion > common.GetSavedVersion(c), check.Equals, true,
```

Comment:
As a general hint, although go-check's assertions are often very handy, there's also a common (as in, not just you) tendency to overuse assertions for things that plain code would work well.

For example, this would work quite well here, and lose no additional information since the code is comparing a boolean to a boolean:
```
if currentVersion <= common.GetSavedVersion(c) {
        c.Fatalf("Rebooted to the wrong version: %d", currentVersion)
}
```